src/views/Setup.vue

```diff
@@ -3,9 +3,16 @@ import { ref } from 'vue';
 import { useRouter } from 'vue-router';
 
 const router = useRouter();
+const tlds = ['.eth', '.xyz', '.com', '.org', '.io', '.app', '.art'];
 
 const id = ref('');
 
+function isDisabledTLD(id) {
+  return !tlds.some(function (tlds) {
+    return id.endsWith(tlds);
+  });
```

Comment:
This could be simplified like this
```suggestion
  return tlds.some(tlds => id.endsWith(tlds));
```